app/models/audit/AuditTaskTable.scala

```diff
@@ -335,10 +335,11 @@ object AuditTaskTable {
     val timestamp: Timestamp = new Timestamp(Calendar.getInstance(TimeZone.getTimeZone("UTC")).getTime.getTime)
 
     val completedTasks = for {
-      u <- users.filter(_.username === user.toString)
-      at <- auditTasks if at.userId === u.userId
+      u <- users.filter(_.userId === user.toString)
```

Comment:
This was changed because the UUID toString method actually returns the userId, not the username.